src/core/qgsfiledownloader.h

```diff
@@ -79,6 +79,9 @@ class GUI_EXPORT QgsFileDownloader : public QObject
      */
     void onDownloadCanceled();
 
+    //! Called to start the download
```

Comment:
Can the public slot `onDownloadCanceled` be renamed to `cancelDownload` (imperative)?